papyrus/tests/test_geo_interface.py

```diff
@@ -236,8 +242,11 @@ def test_geo_interface_declarative_no_feature_as_base(self):
     def _test_geo_interface_no_feature(self, mapped_class):
         from papyrus.geojsonencoder import dumps
         obj = mapped_class()
-        json = dumps(obj)
-        self.assertEqual(json, '{"geometry": null, "type": "Feature", "id": null, "properties": {"text": null, "children": [], "child": null}}')  # NOQA
+        obj_json = dumps(obj)
+        json_parsed = json.loads(obj_json)
+        if PY == 2:
+            json_parsed['id'] = None
```

Comment:
Maybe. If I launch the tests on the master branch against python2, 2 tests fails because they lack the id attribute:
```
======================================================================
FAIL: test_geo_interface_declarative_no_feature (papyrus.tests.test_geo_interface.GeoInterfaceTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenselme/projects/papyrus/papyrus/tests/test_geo_interface.py", line 230, in test_geo_interface_declarative_no_feature
    self._test_geo_interface_no_feature(mapped_class)
  File "/home/jenselme/projects/papyrus/papyrus/tests/test_geo_interface.py", line 240, in _test_geo_interface_no_feature
    self.assertEqual(json, '{"geometry": null, "type": "Feature", "id": null, "properties": {"text": null, "children": [], "child": null}}')  # NOQA
AssertionError: '{"geometry": null, "type": "Feature", "properties": {"text": null, "children": [], "child": null}}' != '{"geometry": null, "type": "Feature", "id": null, "properties": {"text": null, "children": [], "child": null}}'

======================================================================
FAIL: test_geo_interface_declarative_no_feature_as_base (papyrus.tests.test_geo_interface.GeoInterfaceTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenselme/projects/papyrus/papyrus/tests/test_geo_interface.py", line 234, in test_geo_interface_declarative_no_feature_as_base
    self._test_geo_interface_no_feature(mapped_class)
  File "/home/jenselme/projects/papyrus/papyrus/tests/test_geo_interface.py", line 240, in _test_geo_interface_no_feature
    self.assertEqual(json, '{"geometry": null, "type": "Feature", "id": null, "properties": {"text": null, "children": [], "child": null}}')  # NOQA
AssertionError: '{"geometry": null, "type": "Feature", "properties": {"text": null, "children": [], "child": null}}' != '{"geometry": null, "type": "Feature", "id": null, "properties": {"text": null, "children": [], "child": null}}'
```